modules/libcom/src/osi/os/WIN32/osdThread.c

```diff
@@ -12,6 +12,16 @@
  * Author: Jeff Hill
  */
 
+/* pull in _WIN32_WINNT* definitions, and hopefully nothing else */
+#include <sdkddkver.h>
+
```

Comment:
`sdkddkver.h` will define _WIN32_WINNT if it is not already defined on both Visual studio and mingw, though they differ in their choice of value.  So I believe the code below preferring sdk8 will never get executed

Reply:
@FreddieAkeroyd You are of course correct...